app/services/scholars_archive/admin_set_select_service.rb

```diff
@@ -0,0 +1,9 @@
+module ScholarsArchive
+  class AdminSetSelectService
+    def self.select(model, select_options)
+      admin_set_name = YAML.load(File.read("config/admin_set_map.yml"))[model]
+      mapped_admin_set = select_options.select{ |option| option.first == admin_set_name }.flatten
```

Comment:
#casecmp seems safe because it always ends up that configuration files and string comparison ends up being code smell. 


```
mapped_admin_set = select_options.find { |o| option.first.casecmp(admin_set_name).zero? }
[ mapped_admin_set || ENV["SCHOLARSARCHIVE_DEFAULT_ADMIN_SET"] ]
```